rbc/request_internal_test.go

```diff
@@ -0,0 +1,129 @@
+package rbc
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestValReqRepository_Find(t *testing.T) {
+	valReqList, _ := NewValReqRepository()
+	val := ValRequest{nil, nil, nil}
+	valReqList.Save("ID1", &val)
+	if valReqList.Find("ID1") != &val {
+		t.Fatalf("request %v is not found.", valReqList.recv["ID1"])
+	}
+}
+func TestValReqRepository_FindAll(t *testing.T) {
+	valReqList, _ := NewValReqRepository()
+	val := ValRequest{nil, nil, nil}
+	valReqList.Save("ID1", &val)
+	valReqList.Save("ID2", &val)
+	if len(valReqList.FindAll()) != 2 {
+		t.Fatalf("request %v,%v is not found all.", valReqList.recv["ID1"], valReqList.recv["ID2"])
+	}
+	for _, request := range valReqList.FindAll() {
+		valReq, _ := request.(*ValRequest)
+		if !reflect.DeepEqual(valReq, &val) {
+			t.Fatalf("request %v,%v is not found all.", valReqList.recv["ID1"], valReqList.recv["ID2"])
+		}
+	}
+}
+func TestValReqRepository_Save(t *testing.T) {
+	valReqList, _ := NewValReqRepository()
+	val := ValRequest{nil, nil, nil}
+	valReqList.Save("ID1", &val)
+	_, ok := valReqList.recv["ID1"]
+	if !ok {
+		t.Fatalf("request %v is not saved", valReqList.recv["ID1"])
+	}
+}
+func TestValReqRepository_NewValReqRepository(t *testing.T) {
+	_, err := NewValReqRepository()
+	if err != nil {
+		t.Fatalf("val request repository is not created.")
+	}
+}
+func TestNewEchoReqRepository(t *testing.T) {
+	_, err := NewEchoReqRepository()
+	if err != nil {
+		t.Fatalf("echo requst repository is not created.")
+	}
+}
+func TestEchoReqRepository_Find(t *testing.T) {
+	echoReqList, _ := NewEchoReqRepository()
+	val := ValRequest{nil, nil, nil}
+	echo := EchoRequest{ val}
+	echoReqList.Save("ID1", &echo)
+	if echoReqList.Find("ID1") != &echo {
+		t.Fatalf("request %v is not found.", echoReqList.recv["ID1"])
```

Comment:
How about marking a value on fatal message you got (incorrect value) ? Maybe it's not difficult to modify!

Reply:
Fatal messages following this part too, please. I think other parts are good!